src/utils/workflow_utils/src/workflow_utils.c

```diff
@@ -1804,57 +1804,86 @@ bool workflow_set_sandbox(ADUC_WorkflowHandle handle, const char* sandbox)
     return true;
 }
 
-// Workfolder =  [root.sandboxfolder]  "/"  ( [parent.workfolder | parent.id]  "/" )+  [handle.workfolder | handle.id]
-char* workflow_get_workfolder(const ADUC_WorkflowHandle handle)
+// Returns the download base directory
```

Comment:
download base directory, or workflow base directory?  this line and the method name conflict.

Reply:
This should be addressed in a newer commit